src/condor_negotiator.V6/matchmaker.cpp

```diff
@@ -1907,7 +1907,7 @@ Matchmaker::forwardAccountingData(std::set<std::string> &names) {
 				// will be zero.  Don't include those submitters.
 
 				if (updateAd.LookupInteger("ResourcesUsed", resUsed)) {
-					cl->sendUpdates(UPDATE_ACCOUNTING_AD, &updateAd, NULL, false);
+					daemonCore->sendUpdates(UPDATE_ACCOUNTING_AD, &updateAd, NULL, false);
```

Comment:
If we're reducing the use of cl here, let's get rid of it entirely. The two other uses in this function can just call daemonCore->getCollectorList() directly.

Reply:
The commit addressing this looks good.